### ISSUE TITLE
Minimise to tray for Windows wallet GUI client.

### ISSUE THREAD

**Issue by Nex-Coder** [BeamMW/beam#1750]
**Feature title**
Minimise to tray for Windows wallet client.

**Describe the current situation we have**
Currently, as far as I can tell, there is no way to hide the GUI wallet from the Windows taskbar. You can hide it from viewing on screen but it will still remain on the taskbar. Instead, a nice quality of life addition would be to enable hide to tray when clicking the X button foe example.

**Describe the solution you'd like**
If enabled in the settings, the exit (X) button should hide to tray instead of closing the program. Then users could close the program by right clicking the tray icon for the wallet and selecting exit.

**Describe alternatives you've considered (if relevant)**
Another approach could be instead of using the X button, instead replace the minimise button functionality with hide to tray instead. Just like before expect the X button would remain as is.

**Additional context**
This is an example of a tray icon with an exit option (Discord's tray icon).
<img src="https://i.gyazo.com/1bac02e522098b8f6a58ad1c59621952.png">

Also I would just like to give a big thank you for all the work you guys have done! I'm a big fan.
I know this request is very trival, its just something I personally would like to see! Helps with decluttering the task bar when working on other stuff <3

**Comment by gingervik** [comment]
@sasha-abramovich please, take a look. maybe we can do it?

**Comment by sasha-abramovich** [comment]
@gingervik definitely, we had this idea before (https://github.com/BeamMW/beam/issues/374, https://github.com/BeamMW/beam/issues/27), maybe now it's the right time to make it happen:
